backend/src/main/java/de/jmpsoftware/backend/controller/AuthController.java

```diff
@@ -42,21 +43,22 @@ public AuthController(AuthenticationManager authenticationManager, JWTUtils jwtS
     }
 
     @PostMapping(path = "/login")
-    public String login(@RequestBody LoginData data) {
+    public ResponseEntity<String> login(@RequestBody LoginData data) {
```

Comment:
I prefer return the object itself here and throwing a RsponseStatusException or adding a ControllerAdvisor to map the Excptions to the correct http status